services/worker/src/worker/config.py

```diff
@@ -259,6 +259,22 @@ def from_env(cls) -> "ConfigNamesConfig":
             )
 
 
+SPLIT_NAMES_MAX_NUMBER = 30
```

Comment:
frankly, I would hardcode it

Reply:
I like that it's at the same place as other constants.

But yea having a config is overkill, we can change that later